homeassistant/components/light/zwave.py

```diff
@@ -63,6 +63,17 @@ def brightness_state(value):
     return 0, STATE_OFF
 
 
+def byte_to_zwave_brightness(value):
+    """
+    Converts brightness in 0-255 scale to 0-99 scale.
+
+    `value` -- (int) Brightness byte value from 0-255.
+    """
+    if value > 0:
+        return max(1, int((value / 255) * 99))
+    return 0
```

Comment:
If the value is 0, shouldn't we just return `None` (there is no brightness) ?

Reply:
@balloob The brightness value as it was used in the original code was an integer in the range of 0 to 99, with 255 representing a special case. I wanted to stay as close to this behavior as possible to minimize the risk of regressing existing code.

Reply:
We should keep it as returning zero, since this is the brightness that we send to the ZWave device, not the brightness value for hass.